.github/workflows/node-test.yaml

```diff
@@ -116,8 +116,19 @@ jobs:
       uses: ./.github/tmp/post-checkout-steps
 
 
+    - name: Prepare install script
+      id: prepare-install-script
+      shell: bash
+      run: |
+        if [ -f package-lock.json ] || [ -f npm-shrinkwrap.json ]
+        then
+          echo "::set-output name=install-command::npm ci"
+        else
+          echo "::set-output name=install-command::npm install"
```

Comment:
What about yarn.lock?

Reply:
`yarn` support not yet implemented - #6 - it needs a little bit more thought.

Reply:
`npm install` and `npm ci` both read `yarn.lock` automatically in npm 7+.

Reply:
You think we should run `npm ci` when `yarn.lock` is present (until #6 is done)?

Reply:
Yes, that would make the most sense.

Reply:
https://github.com/pkgjs/action/pull/37/commits/4b7e44d3e150f8299e5f14e88d0fa0e18e74121c